crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -131,6 +131,12 @@ fn main() -> Result<()> {
     );
     // TODO (JF): Remove this when releasing 0.3.8
     opts.optflag("", "ddsa-runtime", "(deprecated)");
+    opts.optopt(
+        "",
+        "max-rule-time",
```

Comment:
said previously, put a name that is meaningful **AND** contains the UNIT

`rule-timeout-ms`